shims/0.23/src/main/java/org/apache/hadoop/hive/shims/Hadoop23Shims.java

```diff
@@ -1160,19 +1160,22 @@ public Boolean run() throws Exception {
 
   @Override
   public boolean runDistCp(List<Path> srcPaths, Path dst, Configuration conf) throws IOException {
```

Comment:
please add a test. Rest looks fine.

Reply:
Thanks, I have updated the PR with the test case.